source/content.css

```diff
@@ -74,3 +74,11 @@ Dashboard
 	float: unset !important;
 	margin: 0 auto !important;
 }
+
+/* Prevent DM Modal from closing when clicking the overlay */
+#dm_dialog {
+	pointer-events: none;
+}
+#dm_dialog-dialog {
+	pointer-events: auto;
```

Comment:
Using `unset` is a better way to reset to the default value.

Reply:
Will change, update the README right away.

Reply:
Actually, I was wrong. `unset` doesn't work in this case.